tests/vmlifecycle_test.go

```diff
@@ -159,6 +159,32 @@ var _ = Describe("Vmlifecycle", func() {
 			})
 		})
 
+		Context("with boot order", func() {
+			It("should be able to boot from 'middle' disk", func() {
+				By("defining a VM with an Alpine disk")
+				vm = tests.NewRandomVMWithEphemeralDisk(tests.RegistryDiskFor(tests.RegistryDiskAlpine))
+				By("adding a Cirros Disk with boot order 1")
+				tests.AddEphemeralDiskWithBootOrder(vm, "disk1", "virtio", tests.RegistryDiskFor(tests.RegistryDiskCirros), 1)
+				By("adding a Fedora Disk")
+				tests.AddEphemeralDisk(vm, "disk2", "virtio", tests.RegistryDiskFor(tests.RegistryDiskFedora))
+
+				By("starting VM")
+				vm.Spec.Hostname = "cirros"
+				obj, err := virtClient.RestClient().Post().Resource("virtualmachines").Namespace(tests.NamespaceTestDefault).Body(vm).Do().Get()
```

Comment:
You can use `vm, err = virtclient.VM(tests.NamespaceTestDefault).Create(vm)`.

Reply:
Ok, I'll make this change. I was following the existing patterns.